build-cmd.sh

```diff
@@ -50,7 +50,7 @@ apply_patch()
     local patch="$1"
     local path="$2"
     echo "Applying $patch..."
-    git apply --check --reverse --directory="$path" "$patch" || git apply --directory="$path" "$patch"
+    git apply --check --directory="$path" "$patch" || git apply --directory="$path" "$patch"
```

Comment:
I believe the intent of this line was to do nothing if the patch had been accepted upstream.  we probably do not want to change that